src/middlewared/middlewared/plugins/vm.py

```diff
@@ -131,18 +131,19 @@ def __init__(self, manager, vm):
 
                 vnc_resolution = device['attributes'].get('vnc_resolution', None)
                 vnc_port = int(device['attributes'].get('vnc_port', 5900 + self.vm['id']))
+                vnc_bind = device['attributes'].get('vnc_bind', '0.0.0.0')
 
```

Comment:
Shoudn't we make sur the bind IP is still available?

e.g. add IP, use for bind and later on remove it